chain-spec-generator/src/system_parachain_specs.rs

```diff
@@ -61,16 +74,22 @@ pub fn invulnerables_asset_hub_paseo() -> Vec<(AccountId, AssetHubPolkadotAuraId
 ///
 /// The input must be a tuple of individual keys (a single arg for now since we have just one key).
 pub fn asset_hub_paseo_session_keys(
-	keys: AssetHubPolkadotAuraId,
+	keys: AuraId,
 ) -> asset_hub_paseo_runtime::SessionKeys {
 	asset_hub_paseo_runtime::SessionKeys { aura: keys }
 }
 
+pub fn ah_endowed_accounts() -> Vec<AccountId> {
+	invulnerables_asset_hub_paseo().iter().map(
```

Comment:
@al3mart do we want to fund the collators ? is this required to produce blocks during launch?